docs/src/longruns.md

```diff
@@ -13,6 +18,11 @@ longrun_bw_rhoe_equil_highres
 Moist baroclinic wave. Test the moist dycore initial value problem.
 ```
 ```
+longrun_bw_rhoe_equil_veryhighres
```

Comment:
Can we please change the name from `veryhighres` to `<xx>km` or something similar?